requirements.txt

```
@@ -1,13 +1,49 @@
-Django~=2.2
-
-# postgresql
-psycopg2==2.8.3
-
-# image field 
-pillow==6.0.0
-
-# formatting
+appdirs==1.4.3
+atomicwrites==1.3.0
+attrs==19.1.0
 black==19.3b0
-
-# Google Cloud 
-django-storages[google]
+cachetools==3.1.1
+certifi==2019.6.16
+chardet==3.0.4
+Click==7.0
+coverage==4.5.3
+Django==2.2.2
+django-cors-headers==3.0.2
+django-extensions==2.1.9
+django-filter==2.1.0
+django-oauth-toolkit==1.2.0
+django-storages==1.7.1
```

This one's needed for the uploads to Google Cloud Storage
```suggestion
django-storages==1.7.1
django-storages[google]
```